test/spec.yml

```diff
@@ -68,7 +68,7 @@
             If you don't have a key, head over to https://binaris.com to request one
             *? API Key: *
             *Authentication Succeeded*
-              (use "bn create" to create a template function in your current directory)
+              (use "bn create <runtime> <function>" to create a template function in your current directory)
```

Comment:
IMO this is not clear. either have a working line such as `bn create node8 hello`, or `bn create node8|python2|pypy myFunction`, or instruct to look at the help of the next step, such as `bn create -help`. Using `<runtime>` is not helpful for a user who does not know which runtimes we support. Either way it has to be simple